spec/default_spec.rb

```diff
@@ -12,14 +12,111 @@
       end
 
       case pltfrm
-      when CENTOS_6_OPTS
-        it do
-          expect(chef_run).to create_template('/etc/sysconfig/yum-cron')
-        end
       when CENTOS_7_OPTS
+        describe 'template /etc/yum/yum-cron.conf' do
+          let(:file) { chef_run.template('/etc/yum/yum-cron.conf') }
+          [
+            /update_cmd = default/,
```

Comment:
Mind making the regex more strict doing something like ``/^update_cmd = default$/``? This would ensure that it starts and ends with this exactly. Please fix all the remaining matches as well (unless that match is actually incorrect).